scripts/reset-dockstoretestuser4-resources.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
```

Comment:
A lot of our scripts use some combination of:
```
set -o errexit
set -o pipefail
set -o nounset
set -o xtrace
```
If you find that any of that behavior might be desirable, please add as appropriate.

Reply:
Agreed on most, but will need to avoid `set -o xtrace` though due to how the token is used in the curl calls below. 
https://circleci.com/docs/env-vars/#secrets-masking has more details

Reply:
I added echo commands instead of `set -o xtrace` to document the steps. Should I add more details?

Reply:
No, I think that is fine. There are ways of passing environment variables around without having them show up on the command-line. For example a pgpass file with postgress. I think that may be a good follow-up ticket to create now and can tackle if we run into issues or need to improve this script.